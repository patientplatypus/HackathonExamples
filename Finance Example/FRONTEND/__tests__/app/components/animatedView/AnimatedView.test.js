// @flow weak

import React            from 'react';
import AnimatedView     from '../../../../src/app/components/animatedView/AnimatedView';
import renderer         from 'react-test-renderer'; // needed both for snpashot testing but also to prevent errors from enzyme
import { 
  MemoryRouter
}                       from 'react-router';

describe('AnimatedView component', () => {
  // we need MemoryRouter to set an history for children that would use Link from react-router
  it('renders as expected', () => {
    const component = renderer.create(
      <div>
        <MemoryRouter>
          <AnimatedView>
            <p>
              An animated view content
            </p>
          </AnimatedView>
        </MemoryRouter>
      </div>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
