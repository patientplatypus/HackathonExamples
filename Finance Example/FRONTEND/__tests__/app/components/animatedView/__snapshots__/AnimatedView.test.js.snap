// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedView component renders as expected 1`] = `
<div>
  <section
    className="content view-enter"
  >
    <p>
      An animated view content
    </p>
  </section>
</div>
`;
